README.md

```diff
@@ -117,6 +117,25 @@ export dbConnection=...
 delete_db_scxa_marker_genes.sh
 ```
 
+## `exp_design` and `exp_design_column` tables
+
+### Load data
+Run `bin/load_exp_design.sh`. It requires the following environment variables:
+
+| Variable name         | Description                                                                                                                                                                                                                                  |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `CONDENSED_SDRF_FILE` | Path of the condensed SDRF file of the experiment being loaded                                                                                                                                                                               |
```

Comment:
```suggestion
| `CONDENSED_SDRF_TSV ` | Path of the condensed SDRF file of the experiment being loaded                                                                                                                                                                               |
```

Reply:
I suggested it otherwise and I changed the other variable in my last commit, because I like your variable name better: `CONDENSED_SDRF_FILE `